isofit/test/test_examples.py

```diff
@@ -0,0 +1,103 @@
+import os
```

Comment:
Some of the datasets in `examples/` are now exercised directly by the test suite. Previously these were executed by running a script in the CI system.